topostats/run_topostats.py

```diff
@@ -69,6 +93,20 @@ def create_parser() -> arg.ArgumentParser:
     parser.add_argument(
         "-s", "--save_plots", dest="save_plots", type=bool, required=False, help="Whether to save plots."
     )
+    parser.add_argument(
+        "--type_of_sample",
+        dest="type_of_sample",
+        type=str,
+        required=False,
+        help="Type of sample being processed.",
```

Comment:
Where do we list allowable values?

Reply:
Damn, I meant to remove this as @SylviaWhittle said she had addressed the minimum size based on sample in her thresholding work. I pulled out lots of other changes I'd made but forgot to do so with the arguments, now removed.